rpc/ethereum/backend/backend.go

```diff
@@ -991,13 +991,8 @@ func (e *EVMBackend) SuggestGasTipCap(baseFee *big.Int) (*big.Int, error) {
 // If the London hard fork is not activated at the current height, the query will
 // return nil.
 func (e *EVMBackend) BaseFee(height int64) (*big.Int, error) {
-	cfg := e.ChainConfig()
-	if !cfg.IsLondon(new(big.Int).SetInt64(height)) {
-		return nil, nil
-	}
-
-	// return BaseFee if London hard fork is activated and feemarket is not enabled
-	res, err := e.queryClient.FeeMarket.BaseFee(types.ContextWithHeight(height), &feemarkettypes.QueryBaseFeeRequest{})
```

Comment:
should we remove the FeeMarket client from the JSON-RPC if it's not being used?

Reply:
there's a usage in `SuggestGasTipCap` to fetch feemarket parameters.